src/components/swiper/index.js

```diff
@@ -72,7 +72,9 @@ const Swiper = ( props ) => {
 			swiperWrapperRef.current.addEventListener( ( 'mousedown' ), ( e ) => e.stopPropagation() );
 
 			return () => {
-				swiperWrapperRef.current.removeEventListener( ( 'mousedown' ), ( e ) => e.stopPropagation() );
+				if ( swiperWrapperRef && swiperWrapperRef.current ) {
+					swiperWrapperRef.current.removeEventListener( ( 'mousedown' ), ( e ) => e.stopPropagation() );
+				}
```

Comment:
What about using the optional chaining operator? We can do this instead:
```suggestion
swiperWrapperRef?.current?.removeEventListener( ( 'mousedown' ), ( e ) => e.stopPropagation() );
```
Where the `?.` operator will only execute if exists otherwise is equivalent to `null`. This is basically the same as checking the property first.